python/activator/visit.py

```diff
@@ -84,21 +85,19 @@ def __str__(self):
 @dataclass(frozen=True, kw_only=True)
 class FannedOutVisit(BareVisit):
     # Extra information is added by the fan-out service at USDF.
-    instrument: str             # short name
     detector: int
     private_sndStamp: float     # time of visit publication; TAI in unix seconds
 
     def __str__(self):
         """Return a short string that disambiguates the visit but does not
         include "metadata" fields.
         """
-        return f"(instrument={self.instrument}, groupId={self.groupId}, survey={self.survey} " \
+        return f"(groupId={self.groupId}, survey={self.survey} " \
```

Comment:
I think it's still useful to keep `instrument` as one of the "identifier" fields, even if the activator does almost immediately filter on it...